src/providers/commands.js

```diff
@@ -0,0 +1,163 @@
+                    hints: {
+                        domain: 1
+                    }
+                },
+                {
+                    key: 'sequenceNumber',
+                    name: 'Sequence Number'
+                },
+                {
+                    key: 'comment',
+                    name: 'Comment'
+                },
+                {
+                    key: 'source',
+                    name: 'Source'
+                },
+                {
+                    key: 'queue',
+                    name: 'Queue'
+                },
+                {
+                    key: 'binary',
+                    name: 'Binary'
+                },
+                {
+                    key: 'unprocessedBinary',
+                    name: 'Unprocessed Binary'
+                },
+                {
+                    key: 'Acknowledge_Queued_Status',
+                    name: 'Acknowledge Queued Status'
+                },
+                {
+                    key: 'Acknowledge_Queued_Time',
+                    name: 'Acknowledge Queued Time'
+                },
+                {
+                    key: 'Acknowledge_Released_Status',
+                    name: 'Acknowledge Released Status'
+                },
+                {
+                    key: 'Acknowledge_Released_Time',
+                    name: 'Acknowledge Released Time'
+                },
+                {
+                    key: 'Acknowledge_Sent_Status',
+                    name: 'Acknowledge Sent Status'
+                },
+                {
+                    key: 'Acknowledge_Sent_Time',
+                    name: 'Acknowledge Sent Time'
+                },
+                {
+                    key: 'username',
+                    name: 'Issuer'
+                },
+                {
+                    key: 'origin',
+                    name: 'Origin'
+                },
+                {
+                    key: 'CCSDS_Version',
+                    name: 'CCSDS Version'
+                },
+                {
+                    key: 'CCSDS_Type',
+                    name: 'CCSDS Type'
+                },
+                {
+                    key: 'CCSDS_Sec_Hdr_Flag',
+                    name: 'CCSDS Sec Hdr Flag'
+                },
+                {
+                    key: 'CCSDS_APID',
+                    name: 'CCSDS APID'
+                },
+                {
+                    key: 'CCSDS_Group_Flags',
+                    name: 'CCSDS Group Flags'
+                },
+                {
+                    key: 'Packet_ID',
+                    name: 'Packet ID'
+                }
+            ]
+        }
+    };
+
+    return commandObject;
+}
+
+/**
+ * Convert raw command data from YAMCS to a format which
+ * can be consumed by Open MCT as telemetry.
+ * @param {Object} command
+ * @returns {Object} telemetryPoint
+ */
+export function commandToTelemetryPoint(command) {
```

Comment:
It's confusing, but in the nomenclature we use a telemetry point is the thing producing telemetry (called a "parameter" in Yamcs, a "channel" at JPL, and a "mnemonic" in other environments - everyone uses their own terminology). We refer to the telemetry values themselves as telemetry data (or datum singular). So maybe this should be `commandToTelemetryDatum`?

Reply:
Sounds good to me. I was also dissatisfied with "point". We use it elsewhere as well so I can go ahead and clean this up.